backend/src/graphql/arg/SearchUsersFilters.ts

```diff
@@ -0,0 +1,11 @@
+import { Field, InputType, ObjectType } from 'type-graphql'
+
+@ObjectType()
+@InputType('SearchUsersFiltersInput')
+export default class SearchUsersFilters {
+  @Field(() => Boolean, { nullable: true, defaultValue: null })
+  filterByActivated?: boolean | null
```

Comment:
```suggestion
  filterByActivated?: boolean | null
```

Maybe rename to `byActivated` and as well for similar properties, see open todo in description.

Reply:
Yes. Please do this everywhere.